packages/components/modules/comments/CommentItem/CommentsReplies/index.tsx

```diff
@@ -11,7 +11,6 @@ import { CommentsRepliesProps } from './types'
 
 const CommentsReplies: FC<CommentsRepliesProps> = ({
   target: targetRef,
-  profileId,
```

Comment:
we should also remove the `profileId` from  `CommentsRepliesProps` and from its storybook

Reply:
@anicioalexandre: I could not find it in the storybooks